extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -632,3 +633,160 @@ descendant_tag_list: {
   tag: "VAR"
   tag: "WBR"
 }
+tags: {  # <amp-story-page-attachment>
+  html_format: AMP
+  tag_name: "AMP-STORY-PAGE-ATTACHMENT"
+  mandatory_ancestor: "AMP-STORY-PAGE"
+  descendant_tag_list: "amp-story-page-attachment-allowed-descendants"
+  mandatory_last_child: true
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+}
+descendant_tag_list {
+  # Whitelisting as many components as possible, unless they could result in a
+  # bad UX (ie: modal in a modal with amp-lightbox), or potential security
+  # issues (ie: amp-form).
+  name: "amp-story-page-attachment-allowed-descendants"
+  tag: "A"
+  tag: "ABBR"
+  tag: "ADDRESS"
+  tag: "AMP-3D-GLTF"
+  tag: "AMP-3Q-PLAYER"
+  tag: "AMP-ACCORDION"
+  tag: "AMP-AUDIO"
+  tag: "AMP-BEOPINION"
+  tag: "AMP-BODYMOVIN-ANIMATION"
+  tag: "AMP-BRID-PLAYER"
+  tag: "AMP-BRIGHTCOVE"
+  tag: "AMP-BYSIDE-CONTENT"
+  tag: "AMP-CALL-TRACKING"
+  tag: "AMP-CAROUSEL"
+  tag: "AMP-DAILYMOTION"
+  tag: "AMP-DATE-COUNTDOWN"
+  tag: "AMP-EMBEDLY-CARD"
+  tag: "AMP-FACEBOOK"
+  tag: "AMP-FACEBOOK-COMMENTS"
+  tag: "AMP-FACEBOOK-LIKE"
+  tag: "AMP-FACEBOOK-PAGE"
+  tag: "AMP-FIT-TEXT"
+  tag: "AMP-FX-COLLECTION"
+  tag: "AMP-FX-FLYING-CARPET"
+  tag: "AMP-GFYCAT"
+  tag: "AMP-GFYCAT"
+  tag: "AMP-GIST"
+  tag: "AMP-GIST"
+  tag: "AMP-GOOGLE-DOCUMENT-EMBED"
+  tag: "AMP-GOOGLE-VRVIEW-IMAGE"
+  tag: "AMP-GOOGLE-VRVIEW-IMAGE"
+  tag: "AMP-HULU"
+  tag: "AMP-IMA-VIDEO"
+  tag: "AMP-IMAGE-SLIDER"
+  tag: "AMP-IMG"
+  tag: "AMP-IMGUR"
+  tag: "AMP-INSTAGRAM"
+  tag: "AMP-IZLESENE"
+  tag: "AMP-JWPLAYER"
+  tag: "AMP-KALTURA-PLAYER"
+  tag: "AMP-LIST"
+  tag: "AMP-LIST"
+  tag: "AMP-LIVE-LIST"
+  tag: "AMP-LIVE-LIST"
+  tag: "AMP-MATHML"
+  tag: "AMP-MOWPLAYER"
+  tag: "AMP-NEXXTV-PLAYER"
+  tag: "AMP-O2-PLAYER"
+  tag: "AMP-OOYALA-PLAYER"
+  tag: "AMP-PAN-ZOOM"
+  tag: "AMP-PINTEREST"
+  tag: "AMP-PLAYBUZZ"
+  tag: "AMP-POWR-PLAYER"
+  tag: "AMP-REACH-PLAYER"
+  tag: "AMP-REDDIT"
+  tag: "AMP-RIDDLE-QUIZ"
+  tag: "AMP-SOUNDCLOUD"
+  tag: "AMP-SPRINGBOARD-PLAYER"
+  tag: "AMP-TIMEAGO"
+  tag: "AMP-TWITTER"
+  tag: "AMP-VIDEO"
+  tag: "AMP-VIDEO-IFRAME"
+  tag: "AMP-VIMEO"
+  tag: "AMP-VINE"
+  tag: "AMP-VIQEO-PLAYER"
+  tag: "AMP-VK"
+  tag: "AMP-WISTIA-PLAYER"
+  tag: "AMP-YOTPO"
+  tag: "AMP-YOUTUBE"
+  tag: "ARTICLE"
+  tag: "ASIDE"
+  tag: "B"
+  tag: "BDI"
+  tag: "BDO"
+  tag: "BLOCKQUOTE"
+  tag: "BR"
+  tag: "BUTTON"
+  tag: "CAPTION"
+  tag: "CITE"
```

Comment:
It looks like you skipped a bunch of "Non-conforming features" for HTML in this list, including more widely used tags like CENTER and NOBR, but there are a bunch of them in the validator rules that we allow. I don't see any security or UI reasosn you'd want to. Would it make sense to add them in?

Sorry I didn't notice this before, it's difficult to see what's missing from a list.

Reply:
Thanks! I'm moving this discussion to an email thread so we don't block this PR. We'd want to look into allowing these for amp-story, along with SVGs.